testnets/celestiatestnet2/chain.json

```
@@ -22,19 +22,19 @@
   },
   "codebase": {
     "git_repo": "https://github.com/celestiaorg/celestia-app",
-    "recommended_version": "v1.0.0-rc1",
+    "recommended_version": "v1.0.0-rc5",
```

I would recommend upcoming rc6

```suggestion
    "recommended_version": "v1.0.0-rc6",
```

---

rc6 will be avail soon ™️